src/js/views/views/HallOfFame.js

```diff
@@ -48,15 +48,17 @@ const HallOfFame = ({players}) => {
                 helpers.round(p.careerStats.ewa, 1),
             ],
             classNames: {
-                info: p.statsTids.includes(g.userTid),
+                danger: p.legacyTid === g.userTid,
+                info: p.statsTids.includes(g.userTid) && p.statsTids[p.statsTids.length - 1] !== g.userTid,
```

Comment:
Would these have to be three-way mutually exclusive? e.g.

* red: red and not blue and not green
* blue: not red and blue and not green
* green: not red and not blue and green

I had copied the pattern from teamHistory which looks like:

* red: red
* blue: blue and not green
* green: green

I think there's some implied inequality though, that a player in HoF (red) does not have any tid set at all, therefore they can never be blue or green. So following that, I think it would be enough to set this to:

* danger: player's "best" team is user's tid
* info: user tid is not legacy tid and in statsTids but not the last element
* success: last element of statsTids is user tid and not legacy tid

It's a bit winded, but I think it would be certain to not have two classes applied at once.